classes/04class/exercises/c04-iac04/nimmi89/main.tfvars

```diff
@@ -0,0 +1,4 @@
+vpc_id = vpc-081afcee350babdf5
```

Comment:
This file is duplicated. It should be only inside `terraform-code` folder. Please remove it.

Reply:
Removed the files!